advanced-list/src/app.js

```diff
@@ -12,32 +13,26 @@ function makeRandomColor() {
   return hexColor;
 }
 
-function intent(DOM, tickerAction$) {
-  const tickerRemove$ = tickerAction$.filter(a => a.type === 'remove');
-  const action$ = xs.merge(
-    tickerRemove$
-  );
-  return action$;
+function insertReducer(tickers) {
+  const color$ = xs.periodic(1000)
+    .map(makeRandomColor)
+    .startWith('#000000');
+
+  return tickers.add(color$);
```

Comment:
Here's what I don't understand... what's the signature of tickers.add() ? Isn't it supposed to receive an object of sources? Why isn't it `tickers.add({props$: color$})` and those given sources are merged with the driver sources that were given in line 49?